
# This is a patch for Task-Kensho-0.22.orig to update it to Task-Kensho-0.22
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "Task-Kensho-0.22.orig\Makefile.PL" "Task-Kensho-0.22\Makefile.PL"
Index: ./Makefile.PL
*** ./Makefile.PL	Tue Apr  6 11:56:56 2010
--- ./Makefile.PL	Thu Jul 15 23:05:29 2010
***************
*** 59,65 ****
  feature 'Web Development',
    -default                         => 0,
    'Task::Catalyst'                 => 0,
!   'Template::Toolkit'              => 0,
    'CGI::FormBuilder::Source::Perl' => 0,
    'XML::RSS'                       => 0,
    'XML::Atom'                      => 0,
--- 59,65 ----
  feature 'Web Development',
    -default                         => 0,
    'Task::Catalyst'                 => 0,
!   'Template'                       => 0,
    'CGI::FormBuilder::Source::Perl' => 0,
    'XML::RSS'                       => 0,
    'XML::Atom'                      => 0,
***************
*** 68,74 ****
  
  feature 'Web Crawling',
    -default                      => 0,
!   ' HTTP::Lite'                 => 0,
    'Bundle::LWP'                 => 0,
    'LWP::Simple'                 => 0,
    'WWW::Mechanize'              => 0,
--- 68,74 ----
  
  feature 'Web Crawling',
    -default                      => 0,
!   'HTTP::Lite'                  => 0,
    'Bundle::LWP'                 => 0,
    'LWP::Simple'                 => 0,
    'WWW::Mechanize'              => 0,
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Thu Jul 15 23:06:02 2010
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "Makefile.PL" 3430 1279256729 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Thu Jul 15 23:06:02 2010] ####
#### Patch checksum: 72 2589 11430 ####
#### Checksum: 90 3287 3723 ####
